crates/e2e/src/setup/onchain_components.rs

```diff
@@ -37,6 +37,10 @@ macro_rules! tx {
     };
 }
 
+pub fn to_mwei(base: u32) -> U256 {
+    U256::from(base) * U256::exp10(6)
+}
+
```

Comment:
If I understand your usage of `to_mwei()` correctly this function sounds similar to the existing `to_wei()` but behaves differently.
`to_wei()` takes the number of ETH you want to have and represents it as wei.
`to_mwei()` takes the number of `mwei` you want to have and represents it as wei.
So these functions should probably be called `eth_to_wei()` and `mwei_to_wei()`.

Reply:
Nice catch, will update.